build.ps1

```diff
@@ -274,7 +275,14 @@ try
             Write-Host -ForegroundColor Cyan "Unit Test execution"
             Write-Host
 
-            RunTests "Unit tests" -filterTestCategory "TestCategory=Unit" -framework $framework
+            $testCategory = "TestCategory=Unit";
+
+            if ($skipDeviceStreamTests)
```

Comment:
By definition, unit tests shouldn't hit the service. We don't seem to have other exclusions in this if block, so should this be removed?

Reply:
Or do the other exclusions belong here too?

Reply:
Good catch. Removed this from the unittests section as they hit a fake endpoint (mock tests).